### ISSUE TITLE
Bit boss not resetting on stream

### ISSUE THREAD

**Issue by JasonCessna** [bitboss-ca/png2icns#1]
The local health bar is showing the updated, reset amount, but on stream it is still showing the previous stream's ending bit boss.

I saw that this was an issue in the old repo, but that was moved and is no longer displayed so I apologize for a repeated issue.

**Comment by bitboss-ca** [comment]
I do not believe this is relevant to the repository here.  I'm not sure what you are referring to.  If I'm wrong, please elaborate more on the issue, and provide examples.  Thanks!

**Comment by bitboss-ca** [comment]
Closing as outdated.

**Comment by bitboss-ca** [comment]
Closed.
